sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -167,6 +236,26 @@ class RocksDBSuite extends SparkFunSuite {
     // scalastyle:on line.size.limit
   }
 
+  def withDB[T](
+      remoteDir: String,
+      version: Int = 0,
+      conf: RocksDBConf = RocksDBConf().copy(compactOnCommit = false, minVersionsToRetain = 100),
```

Comment:
Here the `conf` param is always provided - is there usage on using default value?

Reply:
No usage for the default value, delete the default value in the next commit.